engine-comparison/common-harness.sh

```diff
@@ -5,3 +5,47 @@
 # Every "gcloud compute ssh/scp/etc" needs --zone, otherwise throws warning
 GCLOUD_ZONE="us-west1-b"
 
+# Almost definitely fine to be in the public domain
+SERVICE_ACCOUNT="373628893752-compute@developer.gserviceaccount.com"
+
```

Comment:
As far as you, me, and @kcc are concerned, this will be a constant, and is tied to our GCloud project. It feels accurate to define it in the same place as `$GCLOUD_ZONE`, correspondingly it would be something to abstract away when we generalize these scripts.